content/docs/app-developer-guide/export-to-oci-layout.md

```diff
@@ -0,0 +1,151 @@
+    <div class="author">See the <a href="https://github.com/opencontainers/image-spec/blob/main/image-layout.md">specification</a></div>
+</div>
+
+Exporting to OCI layout format is an **experimental** feature available on pack since version X.Y.Z
+
+### 1. Enable experimental feature
+
+Verify your pack version is equal or greater than X.Y.Z
+
+```bash
+pack version
+```
+
+Enable the experimental features on pack
+
+```bash
+pack config experimental true
+```
+
+You can confirm everything is fine, checking the `config.toml` file in your `PACK_HOME` installation folder, for example:
+
+```bash
+cat ~/.pack/config.toml
+experimental = true
+layout-repo-dir = "<$HOME>/.pack/layout-repo"
+```
+
+The configuration shows the experimental mode was **enabled** and a local directory to save images on disk was configured to path `<$HOME>/.pack/layout-repo`.
+
+### 2. Build the app
+
+If you haven't already, please follow the steps to [build an app](/docs/app-developer-guide/build-an-app), once you managed to build a sample application you can try exporting the sample application to disk in OCI layout format. 
+
+The OCI layout feature must be enabled using the convention `oci:<path/to/save/image>` in the `<image-name>` parameter when invoking `pack build`.
+
+For example:
+
+```bash
+pack build oci:sample-app --path samples/apps/java-maven --builder cnbs/sample-builder:bionic
+```
+
+It will save the image in a folder `./sample-app` created in your current directory.
+
+### 3. Check your image
+
+**Congratulations!**
+
+You can verify your application image was saved on disk in a folder called `./sample-app` in your current directory in OCI layout format, for example:
+
+```bash
+tree sample-app
+
+sample-app
+├── blobs
+│ └── sha256
+│     ├── 141bfb0cd434d425bc70edb9e56ea11d07aed76450eb0e73e6110645f251a8d3
+│     ├── 2fa192256ce255c6ea6c1296eadfe2feba8094f40e6aa85e699645caca2e85d8
+│     ├── 5a44e4f7b58d74fe6f92dd7028075c91191128d1e2e7f39846fe061a9a98836e
+│     ├── 72d9f18d70f395ff9bfae4d193077ccea3ca583e3da3dd66f5c84520c0100727
+│     ├── 827746ec7ba80f4e4811b6c9195b6f810fbc2d58a6c9cc337bf0305791f24e97
+│     ├── ad13830c92258c952f25d561d8bf7d9eb58b8a3003960db1502cbda8239130b5
+│     ├── b97b58b190d5f731c879b0f7446a2bd554863b51851e03757199c74dd922ce61
+│     ├── c44222730efa142cd5bedc0babf82a9a07d325494be7f5c3cfde56f43166b65f
+│     ├── e1048fb89c3194a1f0542c0847aa086a7034dd7867c48fe8c93675cf36f90610
+│     ├── f0a30c5bc44742065b1b4ffa95271a39994f05ba7a03dd7e7143d1d3e45fa0b1
+│     └── f9d6350d0c44c0e7165a522155f53181ce8c163a6b8ead1f6baea22d1a8d8a78
+├── index.json
+└── oci-layout  
+
+3 directories, 13 files
+```
+If you are interested on keep playing with the image in  OCI layout format, one tool you can take a look is [umoci](https://umo.ci/) it can help you to create a 
+[runtime bundler](https://github.com/opencontainers/runtime-spec) that can be executed with another tool like [runc](https://github.com/opencontainers/runc)
```

Comment:
There a number of tools here that we might need to explain to end-users.  I *think* we're using `umoci` because it is a strict-OCI compliant tool?  That assembles an image.  I _presume_ that docker can also execute a strict-OCI image?  If so, should we prefer to not introduce a new runtime there?

Reply:
Yeah, I had some doubts about including those tool, maybe we can suppose users know what they can do with the OCI layout format